games/devtest/mods/unittests/raycast.lua

```diff
@@ -0,0 +1,36 @@
+local function raycast_with_pointabilities(start_pos, end_pos, pointabilities)
+	local ray = minetest.raycast(start_pos, end_pos, nil, nil, pointabilities)
+	for hit in ray do
+        if hit.type == "node" then
+            return hit.under
+        end
+	end
+    return nil
+end
+
+local function test_raycast_pointabilities(player, pos1)
+    local pos2 = pos1 + vector.new(0, 0, 1)
+    local pos3 = pos2 + vector.new(0, 0, 1)
+
+    local oldnode1 = core.get_node(pos1)
+    local oldnode2 = core.get_node(pos2)
+    local oldnode3 = core.get_node(pos3)
+    core.swap_node(pos1, {name = "air"})
+    core.swap_node(pos2, {name = "testnodes:not_pointable"})
+    core.swap_node(pos3, {name = "testnodes:pointable"})
+
+    local p = nil
+    assert(raycast_with_pointabilities(pos1, pos3, p) == pos3)
+
+    p = minetest.registered_items["testtools:blocked_pointing_staff"].pointabilities
```

Comment:
s/minetest/core/

Reply:
```suggestion
    p = core.registered_items["testtools:blocked_pointing_staff"].pointabilities
```

Reply:
Updated. "core" vs. "minetest" usage seems to be inconsistent in Devtest.